easybuild/easyconfigs/x/xalt/xalt-2.8.eb

```diff
@@ -0,0 +1,77 @@
+# contributed by Victor Holanda (CSCS)
+
+name = 'xalt'
+version = "2.8"
```

Comment:
I would suggest adding a suffix to make sure that this is our patched version.

Reply:
I am not sure if we should use a suffix, due to the fact the xalt installation is already daint/dom directed and the patch is almost the same as of `Config.py` patch that we apply to get the config. This is just an additional config.

Reply:
@vkarak, what do you think?

Reply:
Fine for me.